dotnet/samples/Demos/Hosting/Agent/ChatWithAgent.Web/AgentCompletionsApiClient.cs

```diff
@@ -14,13 +17,29 @@ internal sealed class AgentCompletionsApiClient(HttpClient httpClient)
     /// <param name="prompt">The prompt.</param>
     /// <param name="cancellationToken">The cancellation token.</param>
     /// <returns>The completion result.</returns>
-    internal async Task<string> CompleteAsync(string prompt, CancellationToken cancellationToken)
+    internal async IAsyncEnumerable<string> CompleteStreamingAsync(string prompt, [EnumeratorCancellation] CancellationToken cancellationToken)
```

Comment:
Although very simplified <string>, we lose the `Usage` and more metadata that can arrive fro the updates, suggest updating the caller to grab the string, or just use the implicit string operator that is already available in the resulting.

```suggestion
    internal async IAsyncEnumerable<string> CompleteStreamingAsync(string prompt, [EnumeratorCancellation] CancellationToken cancellationToken)
```